azure-sql/database/auditing-overview.md

```diff
@@ -42,6 +42,7 @@ You can use SQL Database auditing to:
 ## Auditing limitations
 
 - Enabling auditing on a paused **Azure Synapse** isn't supported. To enable auditing, resume Azure Synapse.
```

Comment:
Team, I think this should be Dedicated SQL Pool since we don't have feature to pause the Azure Synapse. as this will be applicable for Formerly and Synapse Pool